subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/pixels/SubscriptionPixel.kt

```diff
@@ -166,46 +172,56 @@ enum class SubscriptionPixel(
     AUTH_V2_INVALID_REFRESH_TOKEN_DETECTED(
         baseName = "m_privacy-pro_auth_invalid_refresh_token_detected",
         types = setOf(Count, Daily()),
+        removedParameters = setOf(ATB),
     ),
     AUTH_V2_INVALID_REFRESH_TOKEN_SIGNED_OUT(
         baseName = "m_privacy-pro_auth_invalid_refresh_token_signed_out",
         types = setOf(Count, Daily()),
+        removedParameters = setOf(ATB),
     ),
     AUTH_V2_INVALID_REFRESH_TOKEN_RECOVERED(
         baseName = "m_privacy-pro_auth_invalid_refresh_token_recovered",
         types = setOf(Count, Daily()),
+        removedParameters = setOf(ATB),
     ),
     AUTH_V2_MIGRATION_SUCCESS(
         baseName = "m_privacy-pro_auth_v2_migration_success",
         types = setOf(Count, Daily()),
+        removedParameters = setOf(ATB),
     ),
     AUTH_V2_MIGRATION_FAILURE_IO(
         baseName = "m_privacy-pro_auth_v2_migration_failure_io",
         types = setOf(Count, Daily()),
+        removedParameters = setOf(ATB),
     ),
     AUTH_V2_MIGRATION_FAILURE_INVALID_TOKEN(
         baseName = "m_privacy-pro_auth_v2_migration_failure_invalid_token",
         types = setOf(Count, Daily()),
+        removedParameters = setOf(ATB),
     ),
     AUTH_V2_MIGRATION_FAILURE_OTHER(
         baseName = "m_privacy-pro_auth_v2_migration_failure_other",
         types = setOf(Count, Daily()),
+        removedParameters = setOf(ATB),
     ),
     AUTH_V2_TOKEN_VALIDATION_ERROR(
         baseName = "m_privacy-pro_auth_v2_token_validation_error",
         types = setOf(Count, Daily()),
+        removedParameters = setOf(ATB),
     ),
     AUTH_V2_TOKEN_STORE_ERROR(
         baseName = "m_privacy-pro_auth_v2_token_store_error",
         types = setOf(Count, Daily()),
+        removedParameters = setOf(ATB),
     ),
     ;
 
     constructor(
         baseName: String,
         type: PixelType,
         withSuffix: Boolean = true,
-    ) : this(baseName, setOf(type), withSuffix)
+        removedParameters: Set<PixelParameter> = emptySet(),
```

Comment:
@lmac012 I don’t see any pixel having os version btw from the test case. Not sure if that is expected.

Reply:
Yup, we don't send `os_version` param with any of these pixels. This PR doesn't change that.